diff -uraN gnome-boxes-3.38.1/src/vm-configurator.vala omv-gnome-boxes-3.38.1/src/vm-configurator.vala
--- gnome-boxes-3.38.1/src/vm-configurator.vala	2020-10-06 15:59:46.900945700 +0200
+++ omv-gnome-boxes-3.38.1/src/vm-configurator.vala	2020-11-15 18:06:55.383184432 +0100
@@ -463,7 +463,7 @@
             libvirt_name = "sb16";
 
         var model = get_enum_value (libvirt_name, typeof (DomainSoundModel));
-        return_if_fail (model != -1);
+        return_val_if_fail (model != -1, (DomainSoundModel) DomainSoundModel.ICH9);
         return (DomainSoundModel) model;
     }
 
