--- gnome-boxes-3.6.3/configure.ac~	2013-01-14 20:02:46.000000000 +0000
+++ gnome-boxes-3.6.3/configure.ac	2013-06-26 18:46:44.241313673 +0000
@@ -70,7 +70,7 @@
   spice-client-gtk-3.0 >= $SPICE_GTK_MIN_VERSION
   gudev-1.0 >= $GUDEV_MIN_VERSION
   libosinfo-1.0 >= $OSINFO_MIN_VERSION
-  tracker-sparql-0.14 >= $TRACKER_SPARQL
+  tracker-sparql-0.16 >= $TRACKER_SPARQL
   uuid >= $UUID_REQUIRED
   libsoup-2.4 >= $LIBSOUP_REQUIRED
 ])
